tox.ini

```diff
@@ -1,46 +1,14 @@
 [tox]
-envlist = bandit-main, bandit-tests, black, black-check, copyright_check, docs, flake8, liccheck, mypy, py3.8, py3.7, py3.6
-skipsdist = False
-ignore_basepython_conflict = True
+envlist = bandit, black, black-check, copyright_check, docs, flake8, liccheck, mypy, py{3.6,3.7,3.8}
 
 [testenv]
+basepython = python3
+skipsdist = False
+usedevelop = False
+skip_install = False
```

Comment:
by default, we:
- don't skip the creation of the source distribution (the output of `python setup.py sdist`)
- don't use `develop` installation
- don't skip install.

They were the default settings; but migth be good to state them explicitly for self-documentation. Thoughts? O/w if we think it's too verbose we can get rid of them.

Reply:
We could use `develop` in the main testenv to save some time, but better to test also the packaging.

Reply:
I think we can document in the tox.ini as a comment. We should not use develop. I prefer to catch packaging issues early.